### ISSUE TITLE
Endless loop when trying to deploy Wordpress SDL 

### ISSUE THREAD

**Issue by corysturtevant** [akash-network/console-old#93]
**Describe the bug**
Endless loop at pre-flight check when trying to deploy Wordpress SDL from Awesome Akash
https://github.com/akash-network/awesome-akash/blob/master/wordpress/deploy.yaml

**To Reproduce**
1. Go to https://console.akash.network/new-deployment
2. Click on 'Import SDL'
3. Copy and paste SDL from https://github.com/akash-network/awesome-akash/blob/master/wordpress/deploy.yaml
4. Click "Save and Close"
5. Click "Create Deployment"
6. Checking Essentials Pre-flight Check comes back green
7.  Click Next 
8. Page loops clicking next does not progress to next step of deployment

**Expected behavior**
Should click "Next" and proceed with deployment

**Screenshots**
![Akash_Console](https://user-images.githubusercontent.com/52010533/228354573-6ec2963b-697f-4cb3-8894-9b925f1cd66f.png)

https://user-images.githubusercontent.com/52010533/228354769-9d421d7a-1a0f-4396-8c30-81a225d35ba2.mp4

**Desktop (please complete the following information):**
 - OS: macOS 13.2.1 (22D68)
 - Browser Chrome, Brave
 - Version Brave v1.49.132

**Additional context**

**Comment by anilmurty** [comment]
This could be related to https://github.com/akash-network/console/issues/76

**Comment by chainzero** [comment]
If useful and as another example - I also reach this pre-flight check loop when attempting to deploy the SDL in this Discord post.

https://discord.com/channels/747885925232672829/771909909335506955/1090970805514547330

**Comment by anilmurty** [comment]
@yshsaxena - this may have a similar root cause as https://github.com/akash-network/console/issues/94 - not sure if you wanted to attempt to fix this as well?

**Comment by yshsaxena** [comment]
@anilmurty,  yes I will try ;), but This issue we will resolve separately!! as it is more related to #76 , as I found this while debugging the same.

![Screenshot 2023-04-01 145740](https://user-images.githubusercontent.com/45199433/229277962-e7f4d68b-4032-4b0c-b0cb-3470feb84f21.png)

**Comment by anilmurty** [comment]
nice find. Thanks @yshsaxena
